salt/profiles/config/home-deployuser-profiles-docker-compose.yml

```diff
@@ -0,0 +1,40 @@
+version: '3'
+
+services:
+    wsgi:
+        image: elifesciences/profiles:${IMAGE_TAG}
+        command: venv/bin/newrelic-admin run-program venv/bin/uwsgi --ini=/srv/profiles/uwsgi.ini
```

Comment:
turns out this can always be run and New Relic can be turned off with `NEW_RELIC_MONITOR_MODE`